<<<<<<< HEAD
# Service to get whether robot is moving or not

---
bool moving
int64 ret
string msg
=======
# Service to get whether robot is moving or not

---
bool moving
int64 ret
string msg
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
